langlib/lang.boolean/src/main/ballerina/boolean.bal

```diff
@@ -24,6 +24,11 @@ import ballerina/jballerina.java;
 # and also `1` for true and `0` for `false`.
 # This is the inverse of function ``value:toString`` applied to a `boolean`.
 #
+# ```ballerina
+# string str = "true";
+# boolean booleanValue = check boolean:fromString(str);
```

Comment:
I think one example is enough, because user can understand the idea and apply to the other scenarios.

Reply:
I also think one example is enough here, especially since the documentation describes the combinations. Or we will have to add examples for numerous scenarios.

Reply:
Yes. @LakshanWeerasinghe since relevant option is already covered in the description I don't think we need a separate example here. For this context, we are good IMO.